### PR TITLE
report(psi): add lab data summary sentence

### LINKED ISSUE #5882: Include lab data summary sentence in prepareLabData
As requested.

### PR DESCRIPTION
**Pull request by brendankenny**
fixes #5882

### CODE REVIEW

File: lighthouse-core/report/html/renderer/psi.js [hunk 1]
```diff
@@ -57,7 +65,8 @@ function prepareLabData(LHResultJsonString, document) {
   scoreGaugeWrapperEl.removeAttribute('href');
 
   const finalScreenshotDataUri = _getFinalScreenshot(perfCategory);
-  return {scoreGaugeEl, perfCategoryEl, finalScreenshotDataUri};
+  const psiDescription = lhResult.i18n.rendererFormattedStrings.psiDescription;
```
Comment (paulirish) [inline lighthouse-core/report/html/renderer/psi.js]: instead, let's just provide this in the right spot so they dont have to. 

here, `el` is the `perfCategoryEl`:
image


i was writing this but maybe you want to do it differently:

```js
const descriptionText = lhResult.i18n.rendererFormattedStrings.psiDescription;
const descriptionEl = dom.convertMarkdownLinkSnippets(descriptionText);
const catDescriptionEl = dom.find('.lh-category-header__description', perfCategoryEl);
catDescriptionEl.appendChild(descriptionEl);
```

File: lighthouse-core/report/html/renderer/util.js [hunk 2]
```diff
@@ -481,6 +481,9 @@ Util.UIStrings = {
   crcInitialNavigation: 'Initial Navigation',
   /** Label of value shown in the summary of critical request chains. Refers to the total amount of time (milliseconds) of the longest critical path chain/sequence of network requests. Example value: 2310 ms */
   crcLongestDurationLabel: 'Maximum critical path latency:',
+
+  /** Explanation shown to users below performance results to inform them that the test was done with a 3G network connection and to warn them that the numbers they see will likely change slightly the next time they run Lighthouse. 'Lighthouse' becomes link text to additional documentation. */
+  psiDescription: '[Lighthouse](https://developers.google.com/web/tools/lighthouse/) analysis of the current page on emulated 3G. Values are estimated and may vary.',
```
Comment (paulirish) [inline lighthouse-core/report/html/renderer/util.js]: naming tweak: `lsPerformanceCategoryDescription` .. more inline with the ones we have in default-config.js

File: lighthouse-core/report/html/renderer/psi.js (hunk 1 above)
Comment (paulirish) [inline lighthouse-core/report/html/renderer/psi.js]: and we gotta do the markdown conversion :)

File: lighthouse-core/report/html/renderer/util.js (hunk 2 above)
Comment (brendankenny) [reply lighthouse-core/report/html/renderer/util.js]: done

**Comment by brendankenny** [comment]
updated. Now uses the metrics title/description to include these custom strings since that's where we actually want them.

**Pull request merged**